src/wp-includes/pluggable.php

```diff
@@ -91,18 +91,25 @@ function get_userdata( $user_id ) {
 	 *
 	 * @since 2.8.0
 	 * @since 4.4.0 Added 'ID' as an alias of 'id' for the `$field` parameter.
+	 * @since 5.7.0 Returns the global $current_user if it's the user being fetched.
```

Comment:
```suggestion
	 * @since 5.8.0 Returns the global `$current_user` if it's the user being fetched.
```
```suggestion
	 * @since 5.7.0 Returns the global $current_user if it's the user being fetched.
```